src/request/remove-key/RemoveKey.js

```diff
@@ -68,13 +76,24 @@ class RemoveKey {
         }
     }
 
+    /**
+     * @param {string} handler
```

Comment:
Could maybe be typed as `'file' | 'words'` and maybe then `[x: string]: boolean` can be removed from `_exported`'s type declaration.

Reply:
Jup, that worked, good call! :+1: